lib/services/circleci.rb

```diff
@@ -9,7 +9,11 @@ class Service::Circleci < Service
 
   default_events Service::ALL_EVENTS
 
+  string :domain
+  white_list :domain
+
   def receive_event
+    http.ssl[:verify] = false
```

Comment:
I removed the option with an amended sha.  Yes - we should require them to use ssl (or use http if ssl isn't enabled).

The motivation for it was easing initial startup when instance uses self-signed certificate before dealing with ssl certs yet.